src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -239,9 +239,13 @@ private function buildCfpsBlocks(array $endingCfps): array
     /** @param array<mixed> $blocks */
     private function notify(array $blocks): void
     {
-        $this->httpClient->request('POST', $this->webHookUrl, [
-            'headers' => ['Content-type' => 'application/json'],
-            'body' => json_encode($blocks),
-        ]);
+        try {
+            $this->httpClient->request('POST', $this->webHookUrl, [
+                'headers' => ['Content-type' => 'application/json'],
+                'body' => json_encode($blocks),
+            ]);
+        } catch (\Exception $exception) {
+            $this->logger->log('critical', sprintf('An error occured while trying to request Slack : %s', $exception));
```

Comment:
1. use the critical method, it's done for that
2. never cast an exception to a string in a exception message. the context is here for that
3. critical is too high : https://github.com/Seldaek/monolog/blob/main/doc/01-usage.md#log-levels
```suggestion
            $this->logger->error('An error occured while trying to request Slack, ['context' => $exception]);
```